src/index.js

```diff
@@ -106,7 +106,7 @@ class UglifyJsPlugin {
       }
     }
 
-    return warningMessage;
+    return `｢UglifyJs｣ ${warningMessage} in ${file}`;
```

Comment:
Not sure about the brackets. `validateOptions` does not add them.

Reply:
@ooflorent just use `UglifyJs`?

Reply:
Maybe `UglifyJs Plugin` just as here: https://github.com/webpack-contrib/uglifyjs-webpack-plugin/blob/6ab091801ae4e9a0ef59dcab950717a98923a037/src/index.js#L21